build.gradle.kts

```diff
@@ -99,42 +21,5 @@ dependencies {
   runtimeOnly("org.flywaydb:flyway-core:6.3.2")
   runtimeOnly("org.postgresql:postgresql:42.2.12")
 
-  implementation("net.logstash.logback:logstash-logback-encoder:6.3")
-  implementation("com.microsoft.azure:applicationinsights-spring-boot-starter:2.6.0")
-  implementation("com.microsoft.azure:applicationinsights-logging-logback:2.6.0")
-  implementation("com.github.timpeeters:spring-boot-graceful-shutdown:2.2.1")
-
-  implementation("com.fasterxml.jackson.module:jackson-module-kotlin")
-  implementation("com.google.guava:guava:28.2-jre")
-
-  testImplementation("org.springframework.boot:spring-boot-starter-test") {
-    exclude("org.junit.vintage", "junit-vintage-engine")
-  }
   testImplementation("org.springframework.boot:spring-boot-starter-webflux")
-  testImplementation("com.nhaarman.mockitokotlin2:mockito-kotlin:2.2.0")
-}
-
-tasks {
-  test { useJUnitPlatform() }
```

Comment:
@mikehalmamoj I've had to add this in manually in token verification project to get the tests to build.  Shouldn't this be part of the gradle plugin?

Reply:
Well I wrote the Junit 5 test before adding `useJUnitPlatform()` and the test worked without it...  Will investigate further.

Reply:
Try running from command line - the tests don't run for me unless you include the line above

Reply:
`./gradlew clean test`?

Reply:
indeed.  Just been playing around with
```
   id("com.adarshr.test-logger") version "2.0.0"
```
and looks quite good too.  Try running the `clean test` with and without the platform line.

Reply:
Thanks Pete - looks my test that tests the test task is terrible - doesn't find any tests to run but still passes.

Ooh I like the pretty test output, we'll have some of that.